homeassistant/components/media_player/__init__.py

```diff
@@ -115,6 +117,38 @@
     ATTR_INPUT_SOURCE,
 ]
 
+# Service call validation schemas
+MEDIA_PLAYER_SIMPLE_SCHEMA = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+})
+
+MEDIA_PLAYER_MUTE_VOLUME_SCHEMA = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+    vol.Required(ATTR_MEDIA_VOLUME_MUTED): vol.Coerce(bool),
+})
+
+MEDIA_PLAYER_SET_VOLUME_SCHEMA = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+    vol.Required(ATTR_MEDIA_VOLUME_LEVEL): cv.small_float,
+})
+
+MEDIA_PLAYER_MEDIA_SEEK_SCHEMA = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+    vol.Required(ATTR_MEDIA_SEEK_POSITION):
+        vol.All(vol.Coerce(float), vol.Range(min=0)),
+})
+
+MEDIA_PLAYER_PLAY_MEDIA_SCHEMA = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+    vol.Required(ATTR_MEDIA_CONTENT_TYPE): str,
+    vol.Required(ATTR_MEDIA_CONTENT_ID): str,
```

Comment:
We have no strict rules when it comes to content id. It can really be up to the implementation of the media player. Some will reference IDs that are in their media library, others will use URLs to refer online media.

However, you do might want to coerce it to `str`. That way we'll be a bit more accepting.